src/components/Player.js

```diff
@@ -0,0 +1,14 @@
+const Player = () => {
+  const returnedPlayer = {
```

Comment:
zmieniłbym nazwę na samo player z małej litery

Reply:
@BartoszDombrow - nie wiem, dlaczego powiedziałem w biuro, że z dużej. Późno musiało być.
Patryk ma rację. Z małej!

Reply:
Z małej? 
U mnie w komponencie jest z dużej.
@PiotrRynio taka sugestia ode mnie - może zrobimy jakiś plik w readme, w którym wyjaśnimy te konwencje co do nazewnictwa? Można by tam też wrzucić Q&A. Tylko ja się nie chcę tego podjąć, bo już nie wiem, jak powinno być dobrze.

Reply:
Taka konwencja, dość standardowa.
Komponenty duża, 
Ale tutaj jest obiekt/funkcja - one z małej
Jak masz klasę to z dużej

Reply:
to też już zmieniłem, jest z małej